presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -564,8 +564,92 @@ To find the `ROC curve <https://en.wikipedia.org/wiki/Receiver_operating_charact
 
     The thresholds are defined as a sequence whose :math:`j`-th entry is the :math:`j`-th smallest threshold.
 
---------------------------
+Differential Entropy Functions
+-------------------------------
+
+The following functions approximate the binary `differential entropy <https://en.wikipedia.org/wiki/Differential_entropy>`_.
+That is, for a random variable :math:`x`, they approximate
+
+.. math ::
+
+    H(x) = - \int x \log_2\left(f(x)\right) dx,
+
+where :math:`f(x)` is the pdf of :math:`x`.
+
+.. function:: differential_entropy(sample_size, x)
+
+    Returns the approximate log-2 differential entropy from a random variable's sample outcomes. Broadly speaking,
+    the function internally creates a reservoir (see [Black2015]_), then calculates the
+    entropy from the sample results by approximating the derivative of the cumulative distribution
+    (see [Alizadeh2010]_).
+
+    The parameter ``sample_size`` determines the maximal number of reservoir samples. The parameter
```

Comment:
- I'd shorten this to 
```
``sample_size`` is the maximal number of reservoir samples.
```

- this version doesn't have weight parameter; please, remove